src/xtopcom/xvledger/xvblock.h

```diff
@@ -441,7 +443,9 @@ namespace top
             void                set_extend_cert(const std::string & _cert_bin);
 
         private: //m_modified_count not serialized into binary,put here make alignment of this class better
-            uint32_t            m_modified_count;   //count how many times modified since last save,it is 0 if nothing changed
+            mutable uint32_t       m_modified_count;   //count how many times modified since last save,it is 0 if nothing changed
```

Comment:
why m_sign_hash & m_modified_count are mutable?

Reply:
get_hash_to_sign function is const.